.github/workflows/helm_charts_scalar.yml

```diff
@@ -191,11 +191,11 @@ jobs:
 
       - name: Run chart-testing (updated chart)
         if: ${{ github.event_name != 'workflow_dispatch' }}
-        run: ct install --config .github/ct.yaml
+        run: ct install --config .github/ct.yaml --helm-extra-set-args "--wait-for-jobs"
```

Comment:
A question just in case.
So, this `--wait-for-jobs` waits for a job to be completed and doesn't always wait for a specified timeout, right?

Reply:
@feeblefakie  
Thank you for your review!

> So, this `--wait-for-jobs` waits for a job to be completed and doesn't always wait for a specified timeout, right?

Yes. That's right. It doesn't always wait for the specified timeout.
If the job is completed (i.e., the job's status to be `completed`), this command finish and returns `success` at that point.

Reply:
@kota2and3kan Thank you!